src/systems/joint_controller/JointController.cc

```diff
@@ -85,6 +95,71 @@ void JointController::Configure(const Entity &_entity,
     return;
   }
 
+  if (_sdf->HasElement("use_force_commands"))
+  {
+    this->dataPtr->useForceCommands =
+      _sdf->Get<unsigned int>("use_force_commands");
+
+    // PID parameters
+    double p         =  1;
+    double i         =  0.0;
+    double d         =  0.0;
+    double iMax      =  1;
+    double iMin      = -1;
+    double cmdMax    =  1000;
+    double cmdMin    = -1000;
+    double cmdOffset =  0;
+
+    if (_sdf->HasElement("p_gain"))
```

Comment:
Consider simplifying these by using `sdf::Element::Get` with a default value so that the element name doesn't have to be written twice. I think that's less error prone.

Reply:
Good idea. bc7e02c